src/main/Config.h

```diff
@@ -116,6 +116,23 @@ class Config : public std::enable_shared_from_this<Config>
 
     typedef std::shared_ptr<Config> pointer;
 
+    // These test modes should be used in the following contexts:
+    // 1. TESTDB_DEFAULT / TESTDB_BUCKET_DB_VOLATILE: provides the most
+    //    comprehensive end-to-end test, but does not allow arbitrary ledger
```

Comment:
I'm not sure I understand what does 'arbitrary ledger writes' mean. I think it would be more actionable if you explained the actual test implementation implications. IIUC this mode doesn't persist writes between LTXs, but I'm not sure if this is what you're trying to convey with 'arbitrary ledger writes' here.

Reply:
I've updated the comment, as well as changed the test mode names to make this more clear.